hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/BaseCryptoHandler.java

```diff
@@ -51,4 +51,8 @@ protected long getStakedId(
             throw new IllegalStateException("StakedIdOneOfType is not set");
         }
     }
+
+    public static AccountID asAccount(long num) {
```

Comment:
NonNull annotation for result + final for param

Reply:
Done.